docs/reference/configuration.md

```diff
@@ -65,6 +65,8 @@ This reference uses `.` to denote the nesting of values.
     + `database.cosmosdb.endpoint` `(string : "")` - CosmosDB account endpoint, e.g. `https://<account>.documents.azure.com/`.
     + `database.cosmosdb.database` `(string : "")` - CosmosDB database name.
     + `database.cosmosdb.container` `(string : "")` - CosmosDB container name.
+    + `database.cosmosdb.throughput` `(uint : 400)` - The container's RU/s.
```

Comment:
1. how do we control if we lick to configure throughput?
2. prefer to let the user configure the value if choose to use cosmos
3. can we keep the type to int - I assume that invalid values will fail to configure the feature
4. include "CosmosDB..." as part of the parameter description

Reply:
1. Throughput is always configured whether autoscaling or manual mode
2. This is anyway the default for cosmosDB if no configuration provided - using the default in our configuration makes this fact accessible to the user
3. Done
4. Going the other way and removing CosmosDB from all parameter description (redundant + aligning with DyanmoDB description)